### ISSUE TITLE
请问一下Selective Attention 测试部分的代码用diag_part的理由是什么

### ISSUE THREAD

**Issue by dlongq** [thunlp/OpenNRE#120]
您好， 是这样的：

在训练时，Selective Attention 使用了query标签，可以很自然的得到一个shape = [1, 53] 的矩阵。
但是在测试的时候，无法使用query，只能得到shape = [53, 53] 的矩阵。所以我看到您用了tf.diag_part() 在对角线上取值。请问这一步操作有什么数学上的解释嘛

attention_score = tf.nn.softmax(tf.transpose(attention_logit[scope[i][0]:scope[i][1], :]), -1) 
bag_repre_for_each_rel = tf.matmul(attention_score, bag_hidden_mat) 
bag_logit_for_each_rel = __logit__(bag_repre_for_each_rel, rel_tot)
bag_repre.append(bag_repre_for_each_rel)
bag_logit.append(tf.diag_part(tf.nn.softmax(bag_logit_for_each_rel, -1))) 

谢谢！
